### ISSUE TITLE
Can not select a group from drop-down list

### ISSUE THREAD

**Issue by ru2mm** [creecros/Group_assign#7]
Hi
"Assign task to a group on creation or movement" does not work correctly. I can not select a group from drop-down list, because I do not see check box for drop-down list.
I did some groups in Kanboard and added some users in groups.  I typed a group name to the field "Group" in "Define action parameters" window and saved successfully. When I create a new task I get an error: 
Internal Error: SQL Error: SQLSTATE[22P02]: Invalid text representation: 7 ERROR: invalid input syntax for integer: "MYGROUPNAME".
What I am doing wrong?

Kanboard v.1.2.5 
Postgresql10-server-10.5

Thank you.

**Comment by creecros** [comment]
Never tested with postgres, I'll take a look and see if I can work it out for you.

**Comment by creecros** [comment]
tested with 1.2.5 and mysql, works fines.

Can you screen shot the missing dropdown list? Also, did you add a group to the project under the permissions of project?

**Comment by ru2mm** [comment]
Thank you for your answer!
I didn't edit the code. I send you email with screenshots, because I can not attached it here.

**Comment by creecros** [comment]
Try putting the group id # in that field, not sure why it is showing like that though, it should be a dropdown. 
can you list all other plugins you have installed? There may be a conflict, especially if there are any CSS Theme plugins.

**Comment by ru2mm** [comment]
I have put the group id # in that field. The error disappeared, but users from this group did not receive notification by email. I think it is plugins conflict.

**Comment by creecros** [comment]
I never added an EVENT for group change, so there won't be a notification. I did however include an action to email users for a group, so you would have to use that. It would also require an integer for the group, assuming you don't get the dropdown box like the other action.

**Comment by ru2mm** [comment]
Thank you for this solution, but this is not quite the case. I use "Assign the task to a specific user" and it works fine! I want to use "Assign the task to a specific group". It will allow users not to choose whom to assign the task to. I will continue use automatic action "Assign the task to a specific user" for every group members as workaround solution.

**Comment by creecros** [comment]
I had planned to add that feature, so I might get to it, eventually.

**Comment by creecros** [comment]
This plugin is conflicting with it: https://github.com/creecros/AutoSubtasks

They both are overriding the action_creation template, params.php.

I'll see if there is a way around it. Putting on my to do list.

**Comment by creecros** [comment]
For now, you can use the previous version of AutoSubtasks, if you want. It didn't use an override.
https://github.com/creecros/AutoSubtasks/releases/download/0.0.1/AutoSubtasks-0.0.1.zip

**Comment by creecros** [comment]
Updated Autosubtask Plugin to prevent conflict you can download here:
https://github.com/creecros/AutoSubtasks/releases/download/0.0.3/AutoSubtasks-0.0.3.zip

The update will be available on the Kanboard plugin page once the pull request is merged, and then it will show that an "Update is available", if you want to wait.

**Comment by ru2mm** [comment]
Hi
Many thanks for rapid reaction for my question. I tested your solution. I reinstalled AutoSubtask plugin from 0.0.2 version to 0.0.3 version. Problem with drop-down list was resolved. It is super! I did automatic action "Assign the task to a specific group". But group members can not receive email notification after task creation. Group members and Group have permissions "Project Member" in this project. Could you check email notification in your environment?
Thank you.

**Comment by creecros** [comment]
Nothing to check. It's simply not a function of the plugin as of yet.

![image](https://user-images.githubusercontent.com/26339368/45764872-b3109e00-bc01-11e8-8d8f-d6fca53271ed.png)


You'll notice in the Readme, i have it planned for future enhancement, but I am currently working on another plugin: https://github.com/creecros/MetaMagik which I plan to finish before getting back to this plugin.

Currently, you can use the "Send a task by email to assigned group members", which will email the entire group assigned. It functions as a notification, and can also be set to do so during task creation.

**Comment by ru2mm** [comment]
Thank you for information! 
I will wait a stable release of MetaMagik plugin. I think this plugin will very helpful. 
Sorry for disturbing.

**Comment by creecros** [comment]
@ru2mm 
Users will now receive notifications if they are in the assigned group or the multiselect.
https://github.com/creecros/Group_assign/releases/tag/1.3.0
